corehq/apps/data_dictionary/util.py

```diff
@@ -150,7 +150,7 @@ def save_case_property(name, case_type, domain=None, data_type=None,
     except ValidationError as e:
         return str(e)
 
-    if fhir_resource_type:
+    if fhir_resource_type and fhir_resource_prop_path:
```

Comment:
we don't want empty paths to be saved or delete calls as well, if remove_path is True but there is no path (possible via bulk upload)